src/core/style/mixins.scss

```diff
@@ -105,7 +105,15 @@
 // @see https://css-tricks.com/almanac/properties/d/direction/
 
 @mixin rtl($prop, $value, $rtl-value) {
-  #{$prop}: $value;
+  html[dir=ltr] & {
+    #{$prop}: $value;
+  }
+  body[dir=ltr] & {
+    #{$prop}: $value;
+  }
+  html:not([dir]) body:not([dir]) & {
+    #{$prop}: $value;
+  }
```

Comment:
line 108 is to handle html[dir=ltr]
line 111 is to handle body[dir=ltr]
line 114  is to handle case <html><body>... (html and body has not dir) so must be ltr
if do not this we can not reverse ltr to rtl simply for example
margin-left: rem(0.8);
must be 
 @include rtl(margin-left, rem(0.8), null);
@include rtl(margin-right, default_value, rem(0.8));
that default_value depends on other css rule and must be calculated